### ISSUE TITLE
Latest build for linux-64 leads to broken plots in R

### ISSUE THREAD

**Issue by ilevantis** [conda-forge/fontconfig-feedstock#55]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Inside a docker container based on `continuumio/miniconda3:4.10.3` installing `r-ggplot2` and specifying `fontconfig=2.13.96=h8e229c2_1` leads to plots being rendered with all text characters replaced by squares. If  `fontconfig=2.13.96=ha180cfb_0` is specified instead, the issue is not present.

example code to test in each environment.
```
echo -e 'png(\'test.png\')\nggplot2::qplot(1:10)\ndev.off()' > test.R
Rscript test.R
```
**outputs:**
with fontconfig=2.13.96=h8e229c2_1:
![brokentest](https://user-images.githubusercontent.com/10538475/157658836-b4e2412b-adb8-4c0c-954a-f4fb3e229e50.png)

with fontconfig=2.13.96=ha180cfb_0:
![workingtest](https://user-images.githubusercontent.com/10538475/157658839-b1f14c3f-0d3c-4f7b-93f7-7b79259d49f8.png)

### Installed packages

```shell
# ---- BROKEN ENVIRONMENT ----
# packages in environment at /opt/conda:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
_r-mutex                  1.0.1               anacondar_1    conda-forge
binutils_impl_linux-64    2.35.1               h27ae35d_9  
binutils_linux-64         2.35                h67ddf6f_30    conda-forge
brotlipy                  0.7.0           py39h27cfd23_1003  
bwidget                   1.9.14               ha770c72_1    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
ca-certificates           2021.10.8            ha878542_0    conda-forge
cairo                     1.16.0            h6cf1ce9_1008    conda-forge
certifi                   2021.10.8        py39hf3d152e_1    conda-forge
cffi                      1.14.6           py39h400218f_0  
chardet                   4.0.0           py39h06a4308_1003  
conda                     4.11.0           py39hf3d152e_1    conda-forge
conda-package-handling    1.7.3            py39h27cfd23_1  
cryptography              3.4.7            py39hd23ed53_0  
curl                      7.80.0               h7f8727e_0  
expat                     2.4.7                h27087fc_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.13.96              h8e229c2_1    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
freetype                  2.10.4               h0708190_1    conda-forge
fribidi                   1.0.10               h36c2ea0_0    conda-forge
gcc_impl_linux-64         9.3.0               h6df7d76_17  
gcc_linux-64              9.3.0               hf25ea35_30    conda-forge
gettext                   0.19.8.1          h0b5b191_1005    conda-forge
gfortran_impl_linux-64    9.3.0               hc4a2995_19    conda-forge
gfortran_linux-64         9.3.0               hdc58fab_30    conda-forge
graphite2                 1.3.13            h58526e2_1001    conda-forge
gsl                       2.6                  he838d99_2    conda-forge
gxx_impl_linux-64         9.3.0               hbdd7822_17  
gxx_linux-64              9.3.0               h3fbe746_30    conda-forge
harfbuzz                  3.0.0                h83ec7ef_1    conda-forge
icu                       68.2                 h9c3ff4c_0    conda-forge
idna                      2.10               pyhd3eb1b0_0  
jbig                      2.1               h7f98852_2003    conda-forge
jpeg                      9e                   h7f98852_0    conda-forge
kernel-headers_linux-64   2.6.32              he073ed8_15    conda-forge
keyutils                  1.6.1                h166bdaf_0    conda-forge
krb5                      1.19.2               h3790be6_4    conda-forge
ld_impl_linux-64          2.35.1               h7274673_9  
lerc                      2.2.1                h9c3ff4c_0    conda-forge
libarchive                3.5.2                hccf745f_1    conda-forge
libblas                   3.9.0           13_linux64_openblas    conda-forge
libcblas                  3.9.0           13_linux64_openblas    conda-forge
libcurl                   7.80.0               h0b77cf5_0  
libdeflate                1.7                  h7f98852_5    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.3                  he6710b0_2  
libgcc-devel_linux-64     9.3.0               hb95220a_17  
libgcc-ng                 11.2.0              h1d223b6_13    conda-forge
libgfortran-ng            9.4.0               h69a702a_13    conda-forge
libgfortran5              9.4.0               h62347ff_13    conda-forge
libglib                   2.68.4               h3e27bee_0    conda-forge
libgomp                   11.2.0              h1d223b6_13    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
liblapack                 3.9.0           13_linux64_openblas    conda-forge
libnghttp2                1.46.0               hce63b2e_0  
libopenblas               0.3.18          pthreads_h8fe5266_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libsolv                   0.7.19               h780b84a_5    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-devel_linux-64  9.3.0               hf0c5c8d_17  
libstdcxx-ng              11.2.0              he4da1e4_13    conda-forge
libtiff                   4.3.0                hf544144_1    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libwebp-base              1.2.2                h7f98852_1    conda-forge
libxcb                    1.13              h7f98852_1004    conda-forge
libxml2                   2.9.12               h72842e0_0    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
lzo                       2.10              h516909a_1000    conda-forge
make                      4.3                  hd18ef5c_1    conda-forge
mamba                     0.15.3           py39h951de11_0    conda-forge
ncurses                   6.2                  he6710b0_1  
openssl                   1.1.1l               h7f98852_0    conda-forge
pango                     1.48.10              h54213e6_2    conda-forge
pcre                      8.45                 h9c3ff4c_0    conda-forge
pcre2                     10.37                h032f7d1_0    conda-forge
pip                       21.1.3           py39h06a4308_0  
pixman                    0.40.0               h36c2ea0_0    conda-forge
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
pycosat                   0.6.3            py39h27cfd23_0  
pycparser                 2.20                       py_2  
pyopenssl                 20.0.1             pyhd3eb1b0_1  
pysocks                   1.7.1            py39h06a4308_0  
python                    3.9.5                h12debd9_4  
python_abi                3.9                      2_cp39    conda-forge
r-assertthat              0.2.1             r41hc72bb7e_2    conda-forge
r-backports               1.4.1             r41hcfec24a_0    conda-forge
r-base                    4.1.1                hb67fd72_0    conda-forge
r-brio                    1.1.3             r41hcfec24a_0    conda-forge
r-callr                   3.7.0             r41hc72bb7e_0    conda-forge
r-cli                     3.2.0             r41h03ef668_0    conda-forge
r-colorspace              2.0_3             r41h06615bd_0    conda-forge
r-crayon                  1.5.0             r41hc72bb7e_0    conda-forge
r-desc                    1.4.0             r41hc72bb7e_0    conda-forge
r-diffobj                 0.3.5             r41hcfec24a_0    conda-forge
r-digest                  0.6.29            r41h03ef668_0    conda-forge
r-ellipsis                0.3.2             r41hcfec24a_0    conda-forge
r-evaluate                0.15              r41hc72bb7e_0    conda-forge
r-fansi                   1.0.2             r41hcfec24a_0    conda-forge
r-farver                  2.1.0             r41h03ef668_0    conda-forge
r-ggplot2                 3.3.5             r41hc72bb7e_0    conda-forge
r-glue                    1.6.2             r41h06615bd_0    conda-forge
r-gtable                  0.3.0             r41hc72bb7e_3    conda-forge
r-isoband                 0.2.5             r41h03ef668_0    conda-forge
r-jsonlite                1.8.0             r41h06615bd_0    conda-forge
r-labeling                0.4.2             r41hc72bb7e_1    conda-forge
r-lattice                 0.20_45           r41hcfec24a_0    conda-forge
r-lifecycle               1.0.1             r41hc72bb7e_0    conda-forge
r-magrittr                2.0.2             r41hcfec24a_0    conda-forge
r-mass                    7.3_55            r41hcfec24a_0    conda-forge
r-matrix                  1.4_0             r41he454529_0    conda-forge
r-mgcv                    1.8_39            r41h0154571_0    conda-forge
r-munsell                 0.5.0           r41hc72bb7e_1004    conda-forge
r-nlme                    3.1_155           r41h859d828_0    conda-forge
r-pillar                  1.7.0             r41hc72bb7e_0    conda-forge
r-pkgconfig               2.0.3             r41hc72bb7e_1    conda-forge
r-pkgload                 1.2.4             r41h03ef668_0    conda-forge
r-praise                  1.0.0           r41hc72bb7e_1005    conda-forge
r-processx                3.5.2             r41hcfec24a_0    conda-forge
r-ps                      1.6.0             r41hcfec24a_0    conda-forge
r-r6                      2.5.1             r41hc72bb7e_0    conda-forge
r-rcolorbrewer            1.1_2           r41h785f33e_1003    conda-forge
r-rcpp                    1.0.8             r41h03ef668_0    conda-forge
r-rematch2                2.1.2             r41hc72bb7e_1    conda-forge
r-rlang                   0.4.12            r41hcfec24a_0    conda-forge
r-rprojroot               2.0.2             r41hc72bb7e_0    conda-forge
r-rstudioapi              0.13              r41hc72bb7e_0    conda-forge
r-scales                  1.1.1             r41hc72bb7e_0    conda-forge
r-testthat                3.1.2             r41h03ef668_0    conda-forge
r-tibble                  3.1.6             r41hcfec24a_0    conda-forge
r-utf8                    1.2.2             r41hcfec24a_0    conda-forge
r-vctrs                   0.3.8             r41hcfec24a_1    conda-forge
r-viridislite             0.4.0             r41hc72bb7e_0    conda-forge
r-waldo                   0.3.1             r41hc72bb7e_0    conda-forge
r-withr                   2.5.0             r41hc72bb7e_0    conda-forge
readline                  8.1                  h27cfd23_0  
reproc                    14.2.3               h7f98852_0    conda-forge
reproc-cpp                14.2.3               h9c3ff4c_0    conda-forge
requests                  2.25.1             pyhd3eb1b0_0  
ruamel_yaml               0.15.100         py39h27cfd23_0  
sed                       4.8                  he412f7d_0    conda-forge
setuptools                52.0.0           py39h06a4308_0  
six                       1.16.0             pyhd3eb1b0_0  
sqlite                    3.36.0               hc218d9a_0  
sysroot_linux-64          2.12                he073ed8_15    conda-forge
tk                        8.6.10               hbc83047_0  
tktable                   2.10                 hb7b940f_3    conda-forge
tqdm                      4.61.2             pyhd3eb1b0_1  
tzdata                    2021a                h52ac0ba_0  
urllib3                   1.26.6             pyhd3eb1b0_1  
wheel                     0.36.2             pyhd3eb1b0_0  
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libice               1.0.10               h7f98852_0    conda-forge
xorg-libsm                1.2.3             hd9c2040_1000    conda-forge
xorg-libx11               1.7.2                h7f98852_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-libxext              1.3.4                h7f98852_1    conda-forge
xorg-libxrender           0.9.10            h7f98852_1003    conda-forge
xorg-libxt                1.2.1                h7f98852_2    conda-forge
xorg-renderproto          0.11.1            h7f98852_1002    conda-forge
xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xz                        5.2.5                h7b6447c_0  
yaml                      0.2.5                h7b6447c_0  
zlib                      1.2.11               h7b6447c_3  
zstd                      1.5.0                ha4553b6_1  



# ---- WORKING ENVIRONMENT  ----
# packages in environment at /opt/conda:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
_r-mutex                  1.0.1               anacondar_1    conda-forge
binutils_impl_linux-64    2.35.1               h27ae35d_9  
binutils_linux-64         2.35                h67ddf6f_30    conda-forge
brotlipy                  0.7.0           py39h27cfd23_1003  
bwidget                   1.9.14               ha770c72_1    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
ca-certificates           2021.10.8            ha878542_0    conda-forge
cairo                     1.16.0            h6cf1ce9_1008    conda-forge
certifi                   2021.10.8        py39hf3d152e_1    conda-forge
cffi                      1.14.6           py39h400218f_0  
chardet                   4.0.0           py39h06a4308_1003  
conda                     4.11.0           py39hf3d152e_1    conda-forge
conda-package-handling    1.7.3            py39h27cfd23_1  
cryptography              3.4.7            py39hd23ed53_0  
curl                      7.80.0               h7f8727e_0  
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.13.96              ha180cfb_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
freetype                  2.10.4               h0708190_1    conda-forge
fribidi                   1.0.10               h36c2ea0_0    conda-forge
gcc_impl_linux-64         9.3.0               h6df7d76_17  
gcc_linux-64              9.3.0               hf25ea35_30    conda-forge
gettext                   0.19.8.1          h0b5b191_1005    conda-forge
gfortran_impl_linux-64    9.3.0               hc4a2995_19    conda-forge
gfortran_linux-64         9.3.0               hdc58fab_30    conda-forge
graphite2                 1.3.13            h58526e2_1001    conda-forge
gsl                       2.6                  he838d99_2    conda-forge
gxx_impl_linux-64         9.3.0               hbdd7822_17  
gxx_linux-64              9.3.0               h3fbe746_30    conda-forge
harfbuzz                  3.0.0                h83ec7ef_1    conda-forge
icu                       68.2                 h9c3ff4c_0    conda-forge
idna                      2.10               pyhd3eb1b0_0  
jbig                      2.1               h7f98852_2003    conda-forge
jpeg                      9e                   h7f98852_0    conda-forge
kernel-headers_linux-64   2.6.32              he073ed8_15    conda-forge
keyutils                  1.6.1                h166bdaf_0    conda-forge
krb5                      1.19.2               h3790be6_4    conda-forge
ld_impl_linux-64          2.35.1               h7274673_9  
lerc                      2.2.1                h9c3ff4c_0    conda-forge
libarchive                3.5.2                hccf745f_1    conda-forge
libblas                   3.9.0           13_linux64_openblas    conda-forge
libcblas                  3.9.0           13_linux64_openblas    conda-forge
libcurl                   7.80.0               h0b77cf5_0  
libdeflate                1.7                  h7f98852_5    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.3                  he6710b0_2  
libgcc-devel_linux-64     9.3.0               hb95220a_17  
libgcc-ng                 11.2.0              h1d223b6_13    conda-forge
libgfortran-ng            9.4.0               h69a702a_13    conda-forge
libgfortran5              9.4.0               h62347ff_13    conda-forge
libglib                   2.68.4               h3e27bee_0    conda-forge
libgomp                   11.2.0              h1d223b6_13    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
liblapack                 3.9.0           13_linux64_openblas    conda-forge
libnghttp2                1.46.0               hce63b2e_0  
libopenblas               0.3.18          pthreads_h8fe5266_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libsolv                   0.7.19               h780b84a_5    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-devel_linux-64  9.3.0               hf0c5c8d_17  
libstdcxx-ng              11.2.0              he4da1e4_13    conda-forge
libtiff                   4.3.0                hf544144_1    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libwebp-base              1.2.2                h7f98852_1    conda-forge
libxcb                    1.13              h7f98852_1004    conda-forge
libxml2                   2.9.12               h72842e0_0    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
lzo                       2.10              h516909a_1000    conda-forge
make                      4.3                  hd18ef5c_1    conda-forge
mamba                     0.15.3           py39h951de11_0    conda-forge
ncurses                   6.2                  he6710b0_1  
openssl                   1.1.1l               h7f98852_0    conda-forge
pango                     1.48.10              h54213e6_2    conda-forge
pcre                      8.45                 h9c3ff4c_0    conda-forge
pcre2                     10.37                h032f7d1_0    conda-forge
pip                       21.1.3           py39h06a4308_0  
pixman                    0.40.0               h36c2ea0_0    conda-forge
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
pycosat                   0.6.3            py39h27cfd23_0  
pycparser                 2.20                       py_2  
pyopenssl                 20.0.1             pyhd3eb1b0_1  
pysocks                   1.7.1            py39h06a4308_0  
python                    3.9.5                h12debd9_4  
python_abi                3.9                      2_cp39    conda-forge
r-assertthat              0.2.1             r41hc72bb7e_2    conda-forge
r-backports               1.4.1             r41hcfec24a_0    conda-forge
r-base                    4.1.1                hb67fd72_0    conda-forge
r-brio                    1.1.3             r41hcfec24a_0    conda-forge
r-callr                   3.7.0             r41hc72bb7e_0    conda-forge
r-cli                     3.2.0             r41h03ef668_0    conda-forge
r-colorspace              2.0_3             r41h06615bd_0    conda-forge
r-crayon                  1.5.0             r41hc72bb7e_0    conda-forge
r-desc                    1.4.0             r41hc72bb7e_0    conda-forge
r-diffobj                 0.3.5             r41hcfec24a_0    conda-forge
r-digest                  0.6.29            r41h03ef668_0    conda-forge
r-ellipsis                0.3.2             r41hcfec24a_0    conda-forge
r-evaluate                0.15              r41hc72bb7e_0    conda-forge
r-fansi                   1.0.2             r41hcfec24a_0    conda-forge
r-farver                  2.1.0             r41h03ef668_0    conda-forge
r-ggplot2                 3.3.5             r41hc72bb7e_0    conda-forge
r-glue                    1.6.2             r41h06615bd_0    conda-forge
r-gtable                  0.3.0             r41hc72bb7e_3    conda-forge
r-isoband                 0.2.5             r41h03ef668_0    conda-forge
r-jsonlite                1.8.0             r41h06615bd_0    conda-forge
r-labeling                0.4.2             r41hc72bb7e_1    conda-forge
r-lattice                 0.20_45           r41hcfec24a_0    conda-forge
r-lifecycle               1.0.1             r41hc72bb7e_0    conda-forge
r-magrittr                2.0.2             r41hcfec24a_0    conda-forge
r-mass                    7.3_55            r41hcfec24a_0    conda-forge
r-matrix                  1.4_0             r41he454529_0    conda-forge
r-mgcv                    1.8_39            r41h0154571_0    conda-forge
r-munsell                 0.5.0           r41hc72bb7e_1004    conda-forge
r-nlme                    3.1_155           r41h859d828_0    conda-forge
r-pillar                  1.7.0             r41hc72bb7e_0    conda-forge
r-pkgconfig               2.0.3             r41hc72bb7e_1    conda-forge
r-pkgload                 1.2.4             r41h03ef668_0    conda-forge
r-praise                  1.0.0           r41hc72bb7e_1005    conda-forge
r-processx                3.5.2             r41hcfec24a_0    conda-forge
r-ps                      1.6.0             r41hcfec24a_0    conda-forge
r-r6                      2.5.1             r41hc72bb7e_0    conda-forge
r-rcolorbrewer            1.1_2           r41h785f33e_1003    conda-forge
r-rcpp                    1.0.8             r41h03ef668_0    conda-forge
r-rematch2                2.1.2             r41hc72bb7e_1    conda-forge
r-rlang                   0.4.12            r41hcfec24a_0    conda-forge
r-rprojroot               2.0.2             r41hc72bb7e_0    conda-forge
r-rstudioapi              0.13              r41hc72bb7e_0    conda-forge
r-scales                  1.1.1             r41hc72bb7e_0    conda-forge
r-testthat                3.1.2             r41h03ef668_0    conda-forge
r-tibble                  3.1.6             r41hcfec24a_0    conda-forge
r-utf8                    1.2.2             r41hcfec24a_0    conda-forge
r-vctrs                   0.3.8             r41hcfec24a_1    conda-forge
r-viridislite             0.4.0             r41hc72bb7e_0    conda-forge
r-waldo                   0.3.1             r41hc72bb7e_0    conda-forge
r-withr                   2.5.0             r41hc72bb7e_0    conda-forge
readline                  8.1                  h27cfd23_0  
reproc                    14.2.3               h7f98852_0    conda-forge
reproc-cpp                14.2.3               h9c3ff4c_0    conda-forge
requests                  2.25.1             pyhd3eb1b0_0  
ruamel_yaml               0.15.100         py39h27cfd23_0  
sed                       4.8                  he412f7d_0    conda-forge
setuptools                52.0.0           py39h06a4308_0  
six                       1.16.0             pyhd3eb1b0_0  
sqlite                    3.36.0               hc218d9a_0  
sysroot_linux-64          2.12                he073ed8_15    conda-forge
tk                        8.6.10               hbc83047_0  
tktable                   2.10                 hb7b940f_3    conda-forge
tqdm                      4.61.2             pyhd3eb1b0_1  
tzdata                    2021a                h52ac0ba_0  
urllib3                   1.26.6             pyhd3eb1b0_1  
wheel                     0.36.2             pyhd3eb1b0_0  
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libice               1.0.10               h7f98852_0    conda-forge
xorg-libsm                1.2.3             hd9c2040_1000    conda-forge
xorg-libx11               1.7.2                h7f98852_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-libxext              1.3.4                h7f98852_1    conda-forge
xorg-libxrender           0.9.10            h7f98852_1003    conda-forge
xorg-libxt                1.2.1                h7f98852_2    conda-forge
xorg-renderproto          0.11.1            h7f98852_1002    conda-forge
xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xz                        5.2.5                h7b6447c_0  
yaml                      0.2.5                h7b6447c_0  
zlib                      1.2.11               h7b6447c_3  
zstd                      1.5.0                ha4553b6_1
```

### Environment info

```shell
# ---- BROKEN ENVIRONMENT ----
     active environment : base
    active env location : /opt/conda
            shell level : 1
       user config file : /root/.condarc
 populated config files : 
          conda version : 4.11.0
    conda-build version : not installed
         python version : 3.9.5.final.0
       virtual packages : __linux=4.9.51=0
                          __glibc=2.28=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /opt/conda  (writable)
      conda av data dir : /opt/conda/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/conda/pkgs
                          /root/.conda/pkgs
       envs directories : /opt/conda/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/4.11.0 requests/2.25.1 CPython/3.9.5 Linux/4.9.51-10.52.amzn1.x86_64 debian/10 glibc/2.28
                UID:GID : 0:0
             netrc file : None
           offline mode : False



# ---- WORKING ENVIRONMENT  ----
     active environment : base
    active env location : /opt/conda
            shell level : 1
       user config file : /root/.condarc
 populated config files : 
          conda version : 4.11.0
    conda-build version : not installed
         python version : 3.9.5.final.0
       virtual packages : __linux=4.9.51=0
                          __glibc=2.28=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /opt/conda  (writable)
      conda av data dir : /opt/conda/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/conda/pkgs
                          /root/.conda/pkgs
       envs directories : /opt/conda/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/4.11.0 requests/2.25.1 CPython/3.9.5 Linux/4.9.51-10.52.amzn1.x86_64 debian/10 glibc/2.28
                UID:GID : 0:0
             netrc file : None
           offline mode : False
```

**Comment by pkgw** [comment]
Thanks for reporting. The `etc/fonts/fonts.conf` file in the new package has some changes that I think are causing this. I'll work on a new build that restores the previous file structure.

**Comment by pkgw** [comment]
OK, this should be addressed by #56. I want to give someone else time to review the PR but hopefully we can get this fixed swiftly.
